### PR TITLE
Add team gallery

### PR DESCRIPTION
**Pull request by stefanv**
Left to be done:

- [x] Move emeritus core developers to `emeritus` group on GitHub
- [x] Re-render `teams.rst` using the provided `tools/team_list.py`

### CODE REVIEW

**Comment by hmaarrfk** [comment]
Nice nice! Could we add our full names?

**Comment by stefanv** [comment]
They're in there, I just hid them with CSS. Is that generally preferred?

**Comment by hmaarrfk** [comment]
I think many people don't fully get GitHub handles.

Names showing side by side would be nice.

**Comment by jni** [comment]
I need to update my profile pic LOL.

@stefanv the left alignment is hurting my brain. Center them?

I'm agnostic about full vs github name. Happy to go full name.

It would be pretty great if everyone in @scikit-image/core picked a slightly more personalised GH pic, even if it's still anonymous. e.g. can be your favourite dinosaur. Or worm! c.ref. @zpincus. 😜

**Comment by sciunto** [comment]
Awesome!

**Comment by jarrodmillman** [comment]
It would be nice if the images were at the same level on each line.  For instance, the second row of images goes down, up, down, up because some names fit on one line and some fit on two.

**Comment by stefanv** [comment]
@jarrodmillman How's this?

**Comment by emmanuelle** [comment]
Very cool, thanks! @rfezzani @sciunto @soupault @lagru now is the time to see if you're happy with your profile pic :-)

File: team.rst [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+     <a href="https://github.com/sciunto" class="team-member-name">François Boulogne</a>
+     <div class="team-member-handle">@sciunto</div>
+   </div>
+
+
+.. raw:: html
+
+   <div class="team-member">
+     <div class="team-member-photo">
+       <img src="https://avatars1.githubusercontent.com/u/1315589?u=925c7a474919699fc03c39a75876dd0abc0fa790&v=4&s=40"/>
```
Comment (soupault) [inline team.rst:135]: I would prefer to have a photo 😅 . The one from the Google Scholar profile would work - https://scholar.googleusercontent.com/citations?view_op=medium_photo&user=G4gN8BEAAAAJ&citpid=1 .

Comment (emmanuelle) [reply team.rst:135]: do you mind using it on Github? It would be much easier for generating the gallery.

Comment (stefanv) [reply team.rst:135]: @soupault We just use the one you have on GitHub.  Could you update your GitHub profile picture?

Comment (soupault) [reply team.rst:135]: I guess I don't mind 🙂 ... It is updated now.

**Comment by stefanv** [comment]
@soupault A bit brighter might be good, but up to you!

File: team.rst [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+     <a href="https://github.com/sciunto" class="team-member-name">François Boulogne</a>
+     <div class="team-member-handle">@sciunto</div>
+   </div>
+
+
+.. raw:: html
+
+   <div class="team-member">
+     <div class="team-member-photo">
+       <img src="https://avatars2.githubusercontent.com/u/1315589?u=05d88a102b0ab695b824525ea208c0a7e561c2c2&v=4&s=40"/>
```
Comment (soupault) [inline team.rst:135]: @stefanv Good point, thank you!
```suggestion
       <img src="https://avatars2.githubusercontent.com/u/1315589?u=4ef9afb42ea75a6a9a6f6f9afcca0a4114b0c0b5&v=4&s=40"/>
```

File: Makefile [hunk 3]
```diff
@@ -55,3 +55,7 @@ site: html
 # Copy changes to the repo from which they are served
 gh-pages: site
 	python _scripts/gh-pages.py
+
+team:
+	python tools/team_list.py > team.rst
```
Comment (lagru) [inline Makefile:60]: What do you think about (re)creating the `team.rst` with each website build in the CI?

File: tools/team_list.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        profile = api(member['url'])
+
+        print(f"""
+.. raw:: html
+
+   <div class="team-member">
+     <div class="team-member-photo">
+       <img src="{member['avatar_url']}&s=40"/>
+     </div>
+     <a href="https://github.com/{member['login']}" class="team-member-name">{profile['name']}</a>
```
Comment (lagru) [inline tools/team_list.py:43]: Maybe move the image into the `<a>...</a>`-tag as well. Might make the hyperlink a little more touch-friendly/accessible if the CSS allows it.

File: tools/team_list.py [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+def render_team(team):
+    for member in team:
+        profile = api(member['url'])
+
+        print(f"""
+.. raw:: html
+
+   <div class="team-member">
+     <div class="team-member-photo">
+       <img src="{member['avatar_url']}&s=40"/>
```
Comment (lagru) [inline tools/team_list.py:41]:
```suggestion
       <img src="{member['avatar_url']}&s=40" alt="Avatar picture of {profile['name']}."/>
```

Comment (lagru) [reply tools/team_list.py:41]: Also adding a `loading="lazy"` here might be worth it for reducing requests to third parties before rendering the start page?

File: Makefile (hunk 3 above)
Comment (lagru) [reply Makefile:60]: Sorry, my bad. Just noticed that updating the website isn't fully automated anyway. :sweat_smile:

Comment (stefanv) [reply Makefile:60]: @lagru Perfectly fine; we must just set up the tokens.  With me, I mean maybe... some volunteer ;)

Comment (lagru) [reply Makefile:60]: Noted. I'll add it to my idea list. :smile:

**Comment by stefanv** [comment]
Thanks, @lagru, addressed.

I can do a deployment of the website in another PR, then I'll automate the team loading too.

**Comment by stefanv** [comment]
Any other thoughts before this goes in?

**Comment by jni** [comment]
Love it! 🚀

**Comment by rfezzani** [comment]
Thank you @stefanv for this, it's an excellent idea! If you don't mind, I'll keep my avatar as is ;)

**Comment by jni** [comment]
@rfezzani 

> If you don't mind, I'll keep my avatar as is ;)

You've updated it (right?) to something that is not a standard GH anonymous icon, so I'm very happy with it. =)

**Comment by stefanv** [comment]
I think @rfezzani still has the GitHub default icon?

Either way, this is now live on the website.  Thanks all!

**Comment by rfezzani** [comment]
@stefanv, no, it is a ~random gravatar generated long times ago, but not the default GH one :)

**Comment by jni** [comment]
@stefanv you can see Riadh's original GH default in the screenshots at the top. It seems funny to say it but this is much more personal! :joy:

**Pull request merged**